### ISSUE TITLE
Remove connectors at chunk boundaries within same component

### ISSUE THREAD

**Issue by joehagmann** [graph-genome/Schematize#65]
If a single component is split into chunks, there are meaningless connectors at the chunk boundaries. Example covid-19:

![image](https://user-images.githubusercontent.com/28984545/78874512-cb971200-7a4c-11ea-94a7-929f81485e8a.png)

corresponding bin2file:

![image](https://user-images.githubusercontent.com/28984545/78874597-ea95a400-7a4c-11ea-8f23-f44e753f7a20.png)

the chunk boundary in the middle (pan coord 2260) is unnecessary

**Comment by josiahseaman** [comment]
You are correct in thinking this doesn't look good to the user.  For clarification, this is a symptom of a desired feature which was added here: https://github.com/graph-genome/component_segmentation/issues/7 where large components get cut in half to fit in multiple files.  We're going to need some way of detecting if a split is extraneous.

* If the component.last_bin == chunk.last_bin and departures.length == 0 and next_component.arrivals.length == 0
    * then it's an unnecessary split on chunk boundary and should be rendered as continuous, 
    * maybe even change the PangenomeSchematic.js representation to be one long component.
